release-notes/swan-lake-beta4-release-note.md

```diff
@@ -298,6 +298,20 @@ float f13 = 0x1A.0;
 float f14 = 0x1A.0p4;
 ```
 
+- Intervening white spaces are disallowed in the qualified identifier to avoid parsing ambiguity between ternary conditional expression and qualified identifier.
```

Comment:
```suggestion
- Disallowed intervening white spaces in the qualified identifier to avoid parsing an ambiguity between the ternary conditional expression and qualified identifier.
```